runners/katacoda/index.ts

```diff
@@ -228,6 +228,21 @@ export class Katacoda extends Runner {
 
     }
 
+    runCloneRepository(step: Step, command: Command): RunResult {
+
+        let cdCommand = this.changeCurrentDir(path.join("/root", "devonfw", "workspaces", "main"));
+        let workspaceDir = path.join("devonfw", "workspaces", "main");
+        let directoryPath = path.join(workspaceDir, command.parameters[0]).replace(/\\/g, "/");
```

Comment:
You do not need the workspace directory in the directory path for the repo. After the execution of the first cd command the user is already in the workspace directory.